docs/_rpc/server.md

```diff
@@ -18,72 +27,197 @@ For example, the `eth_call` method resides in the `eth` namespace.
 Access to RPC methods can be enabled on a per-namespace basis. Find
 documentation for individual namespaces in the sidebar.
 
-### HTTP Server
-
-To enable the HTTP server, use the `--http` flag.
-
-    geth --http
-
-By default, geth accepts connections from the loopback interface (127.0.0.1).
-The default listening port is 8545. You can customize address and port using the
-`--http.port` and `--http.addr` flags.
+## RPC
 
-    geth --http --http.port 3334
+[RPC (Remote Procedure Calls)][rpc] are programs that execute a procedure in an address
+space other than their own. Address spaces can be separate processes on a single
+machine or different machines. RPC is a client-server interaction where a client sends 
+a request that includes a payload that invokes some method in the server and returns
+the result in a response to the client. The RPC has to be implemented using some transport 
+system - for Geth this transport system can be IPC, HTTP or Websockets. These are individually
+enabled using command line flags when Geth is started.
```

Comment:
Not sure if you want to be this granular here, but I believe IPC is active by default.

Reply:
ah yea good point, thanks.
```suggestion
enabled using command line flags when Geth is started (IPC is enabled by default).
```